src/ralph/lib/openstack/client.py

```diff
@@ -193,3 +201,151 @@ def get_keystone_projects(self):
         else:
             projects_resource = self.keystone_client.tenants
         yield from projects_resource.list()
+
+
+class RalphOpenStackInfrastructureClient:
```

Comment:
Having both `RalphOpenStackInfrastructureClient` and `RalphOpenstackClient` is little confusing. How `RalphOpenStackInfrastructureClient` is different from `RalphOpenstackClient` and which one should I use? We should have doscstrings for this classes describing their purpose and how to use them.